storage/merge_test.go

```diff
@@ -388,17 +387,12 @@ func TestCompactingChunkSeriesMerger(t *testing.T) {
 	// histogramSample returns a histogram that is unique to the ts.
 	histogramSample := func(ts int64) sample {
 		idx := ts + 1
-		return sample{t: ts, h: &histogram.Histogram{
-			Schema:          2,
-			ZeroThreshold:   0.001,
-			ZeroCount:       2 * uint64(idx),
-			Count:           5 * uint64(idx),
-			Sum:             12.34 * float64(idx),
-			PositiveSpans:   []histogram.Span{{Offset: 1, Length: 2}, {Offset: 2, Length: 1}},
-			NegativeSpans:   []histogram.Span{{Offset: 2, Length: 1}, {Offset: 1, Length: 2}},
-			PositiveBuckets: []int64{1 * idx, -1 * idx, 3 * idx},
-			NegativeBuckets: []int64{1 * idx, 2 * idx, 3 * idx},
-		}}
+		return sample{t: ts, h: tsdbutil.GenerateTestHistogram(int(idx))}
+	}
+
+	floatHistogramSample := func(ts int64) sample {
+		idx := float64(ts + 1)
+		return sample{t: ts, fh: tsdbutil.GenerateTestFloatHistogram(int(idx))}
```

Comment:
(and casting to float is not required)
```suggestion
		return sample{t: ts, fh: tsdbutil.GenerateTestFloatHistogram(int(ts+1))}
```